python/res/enkf/config/custom_kw_config.py

```diff
@@ -14,12 +14,14 @@
 #  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
 #  for more details.
 import sys
+import os
```

Comment:
Also remember to clean up imports if needed

Reply:
ok, os